gh-pages/_index.html

```diff
@@ -0,0 +1,472 @@
+<h2 id="plugins">
+  Plugins
+</h2>
+<p>
+  Plugins are NPM packages that extend or modify Blockly's functionality and can be included in any Blockly-based project.
+</p>
+<br>
+<h3 id="fields">
+  Fields
+</h3>
+<p>
+  Field plugins add new kinds of fields that can be incorporated into custom blocks, allowing your users to enter information in different ways. Learn more about how to <a href="https://developers.google.com/blockly/guides/create-custom-blocks/fields/customizing-fields/overview">create and customize fields</a>.
+</p>
+<div class="grid">
+  <a class="card" href="plugins/field-bitmap/test/index.html">
+    <img srcset="media/bitmap-field.png 2x" alt="">
+    <div>
+      <h4>
+        Bitmap Field
+      </h4>
+      <p>
+        A field that lets users input a pixel grid by clicking or tapping.
+      </p>
+    </div>
+  </a>
+  <a class="card" href="plugins/field-date/test/index.html">
+    <img srcset="media/date-field.png 2x" alt="">
+    <div>
+      <h4>
+        Date Field
+      </h4>
+      <p>
+        A field that lets users enter a date using a date picker control.
+      </p>
+    </div>
+  </a>
+  <a class="card" href="plugins/field-colour/test/index.html">
+    <img srcset="media/colour-field.png 2x" alt="">
+    <div>
+      <h4>
+        Colour Field
+      </h4>
+      <p>
+        A field that lets users choose a colour. Although the core Blockly library includes this field, it will soon be removed, so developers should use this plugin instead.
```

Comment:
"soon be removed" -> "removed before the end of 2023"

(so that we don't have to quibble about "soon")

Reply:
Done.